### ISSUE TITLE
Tail Docker Support detects multilines too eagerly

### ISSUE THREAD

**Issue by coders-kitchen** [fluent/fluent-bit#2585]
## Bug Report

**Describe the bug**
We do observe that the tail input plugin with enabled Docker Support, joins log lines to eagerly.
It doesn't respect the given regex for first line detection.

This leads from time to time to the situation that too many logs are collected and 
- the output to graylog fails with too many chunks
- fluentbit consumes 100% CPU

**To Reproduce**
- Rubular link if applicable: https://rubular.com/r/ni64YDTPS1Ea1v
- Example log message if applicable:
```
{"log":"ACCESS [2020-09-25T07:47:03.299Z] \"GET /unleash/api/admin/metrics/feature-toggles HTTP/1.1\" 200 - 0 151 3 2 \"10.244.7.0\" \"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36\" \"686477e2-fb40-447a-9dc9-21334aa816ca\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.09612071Z"}
{"log":"ACCESS [2020-09-25T07:47:05.296Z] \"GET /unleash/api/admin/metrics/feature-toggles HTTP/1.1\" 304 - 0 0 5 3 \"10.244.7.0\" \"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36\" \"e98770d5-836e-48e7-9171-43f74a734a25\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096196769Z"}
{"log":"ACCESS [2020-09-25T07:47:06.030Z] \"GET /unleash/api/client/features HTTP/1.1\" 304 - 0 0 5 4 \"10.244.8.0\" \"whatsappdispatcher\" \"41faa28c-083a-460a-a126-9a9d8e7ed3f4\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096210104Z"}
{"log":"ACCESS [2020-09-25T07:47:07.443Z] \"GET /unleash/api/client/features HTTP/1.1\" 304 - 0 0 6 4 \"10.244.7.0\" \"whatsappdispatcher\" \"c4c38fb5-6553-45e3-a977-eb2cd1ebd9e0\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096246091Z"}
{"log":"ACCESS [2020-09-25T07:47:07.745Z] \"GET /unleash/api/client/features HTTP/1.1\" 304 - 0 0 5 3 \"10.244.7.0\" \"whatsappdispatcher\" \"d6761038-9ab9-4e8f-a558-1491fade3c39\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096256042Z"}
{"log":"ACCESS [2020-09-25T07:47:08.300Z] \"GET /unleash/api/admin/metrics/feature-toggles HTTP/1.1\" 304 - 0 0 3 1 \"10.244.7.0\" \"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36\" \"9b5358aa-7c42-4853-92ac-8f6177f3c19d\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096264898Z"}
{"log":"ACCESS [2020-09-25T07:47:10.297Z] \"GET /unleash/api/admin/metrics/feature-toggles HTTP/1.1\" 304 - 0 0 3 1 \"10.244.7.0\" \"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36\" \"4929e3e5-8146-4832-957f-d6d12cdc3725\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096274135Z"}
{"log":"ACCESS [2020-09-25T07:47:11.030Z] \"GET /unleash/api/client/features HTTP/1.1\" 304 - 0 0 5 3 \"10.244.8.0\" \"whatsappdispatcher\" \"d5da1102-dd20-47cb-8f28-291b1915554e\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096284581Z"}
{"log":"ACCESS [2020-09-25T07:47:12.443Z] \"GET /unleash/api/client/features HTTP/1.1\" 304 - 0 0 5 4 \"10.244.7.0\" \"whatsappdispatcher\" \"6136cb52-56fb-45be-bb4f-31d09e730586\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096293667Z"}
{"log":"ACCESS [2020-09-25T07:47:12.745Z] \"GET /unleash/api/client/features HTTP/1.1\" 304 - 0 0 6 4 \"10.244.7.0\" \"whatsappdispatcher\" \"928911a2-3f3b-4541-b43e-0b09be23c61c\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096302327Z"}
```

Is joined to (stripping out the stream and time information.

ACCESS [2020-09-25T07:47:03.299Z] "GET /unleash/api/admin/metrics/feature-toggles HTTP/1.1" 200 - 0 151 3 2 "10.244.7.0" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36" "686477e2-fb40-447a-9dc9-21334aa816ca" "chat-api-int.tech-on-air.com:30080" "10.245.208.187:4242"
ACCESS [2020-09-25T07:47:05.296Z] "GET /unleash/api/admin/metrics/feature-toggles HTTP/1.1" 304 - 0 0 5 3 "10.244.7.0" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36" "e98770d5-836e-48e7-9171-43f74a734a25" "chat-api-int.tech-on-air.com:30080" "10.245.208.187:4242"
ACCESS [2020-09-25T07:47:06.030Z] "GET /unleash/api/client/features HTTP/1.1" 304 - 0 0 5 4 "10.244.8.0" "whatsappdispatcher" "41faa28c-083a-460a-a126-9a9d8e7ed3f4" "chat-api-int.tech-on-air.com:30080" "10.245.208.187:4242"
ACCESS [2020-09-25T07:47:07.443Z] "GET /unleash/api/client/features HTTP/1.1" 304 - 0 0 6 4 "10.244.7.0" "whatsappdispatcher" "c4c38fb5-6553-45e3-a977-eb2cd1ebd9e0" "chat-api-int.tech-on-air.com:30080" "10.245.208.187:4242"
ACCESS [2020-09-25T07:47:07.745Z] "GET /unleash/api/client/features HTTP/1.1" 304 - 0 0 5 3 "10.244.7.0" "whatsappdispatcher" "d6761038-9ab9-4e8f-a558-1491fade3c39" "chat-api-int.tech-on-air.com:30080" "10.245.208.187:4242"
ACCESS [2020-09-25T07:47:08.300Z] "GET /unleash/api/admin/metrics/feature-toggles HTTP/1.1" 304 - 0 0 3 1 "10.244.7.0" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36" "9b5358aa-7c42-4853-92ac-8f6177f3c19d" "chat-api-int.tech-on-air.com:30080" "10.245.208.187:4242"
ACCESS [2020-09-25T07:47:10.297Z] "GET /unleash/api/admin/metrics/feature-toggles HTTP/1.1" 304 - 0 0 3 1 "10.244.7.0" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36" "4929e3e5-8146-4832-957f-d6d12cdc3725" "chat-api-int.tech-on-air.com:30080" "10.245.208.187:4242"
ACCESS [2020-09-25T07:47:11.030Z] "GET /unleash/api/client/features HTTP/1.1" 304 - 0 0 5 3 "10.244.8.0" "whatsappdispatcher" "d5da1102-dd20-47cb-8f28-291b1915554e" "chat-api-int.tech-on-air.com:30080" "10.245.208.187:4242"
ACCESS [2020-09-25T07:47:12.443Z] "GET /unleash/api/client/features HTTP/1.1" 304 - 0 0 5 4 "10.244.7.0" "whatsappdispatcher" "6136cb52-56fb-45be-bb4f-31d09e730586" "chat-api-int.tech-on-air.com:30080" "10.245.208.187:4242"
ACCESS [2020-09-25T07:47:12.745Z] "GET /unleash/api/client/features HTTP/1.1" 304 - 0 0 6 4 "10.244.7.0" "whatsappdispatcher" "928911a2-3f3b-4541-b43e-0b09be23c61c" "chat-api-int.tech-on-air.com:30080" "10.245.208.187:4242"

- Steps to reproduce the problem:

**Expected behavior**
We do expect the the sample logs are not detected as multiline logs

**Your Environment**

* Version used:
* Configuration:
* Environment name and version (e.g. Kubernetes? What version?):

- Kubernetes: 1.18.8
- FluentBit Docker Image 1.15

* Server type and version:
* Operating System and version:
* Filters and plugins:
    [SERVICE]
        Flush         1
        Log_Level     info
        Daemon        off
        Parsers_File  parsers.conf
        HTTP_Server   On
        HTTP_Listen   0.0.0.0
        HTTP_Port     2020

    [INPUT]
        Name              tail
        Tag               kube.*
        Path              /var/log/containers/*.log
        Exclude_Path      /var/log/containers/wa*whatsapp-instances*.log
        Parser            docker
        DB                /var/log/flb_kube.db
        Mem_Buf_Limit     5MB
        Ignore_Older      3600
        Skip_Long_Lines   On
        Docker_Mode       On
        Docker_Mode_Parser  docker_multiline
        Refresh_Interval  1

    [FILTER]
        Name                kubernetes
        Match               kube.*
        Kube_URL            https://kubernetes.default.svc:443
        Kube_CA_File        /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
        Kube_Token_File     /var/run/secrets/kubernetes.io/serviceaccount/token
        Kube_Tag_Prefix     kube.var.log.containers.
        Merge_Log           On
        Merge_Log_Key       log_processed
        K8S-Logging.Parser  On
        K8S-Logging.Exclude Off

    [FILTER]
        Name record_modifier
        Match *
        Record cluster_name CHAT-API
        Record log_collector fluentbit
    
    [FILTER]
        Name nest
        Match kube.*
        Operation lift
        Nested_under kubernetes

    [FILTER]
        Name nest
        Match kube.*
        Operation lift
        Nested_under labels

    [FILTER]
        Name nest
        Match kube.*
        Operation lift
        Nested_under annotations

    [PARSER]
        Name docker_multiline
        Format regex
        Regex (?<log>^{"log":"\d{4}-\d{2}-\d{2}.*)

    [PARSER]
        Name   json
        Format json
        Time_Key time
        Time_Format %d/%b/%Y:%H:%M:%S %z

    [PARSER]
        Name        docker
        Format      json
        Time_Key    time
        Time_Format %Y-%m-%dT%H:%M:%S.%L
        Time_Keep   On

    [PARSER]
        Name        syslog
        Format      regex
        Regex       ^\<(?<pri>[0-9]+)\>(?<time>[^ ]* {1,2}[^ ]* [^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?(?:[^\:]*\:)? *(?<message>.*)$
        Time_Key    time
        Time_Format %b %d %H:%M:%S   

    [OUTPUT]
        Name                    gelf
        Match                   kube.*
        Host                    <server ip>
        Port                    12201
        Mode                    udp
        Gelf_Short_Message_Key  log

**Additional context**

We want to use the multiline merge capabilities of fluentbit tail plugin to ensure the java stacktraces are not present as single messages, but only one

**Comment by coders-kitchen** [comment]
Some more context:

We removed the `Docker_Mode_Parser  docker_multiline` and apparently the logs are processed line by line (as expected).

**Comment by sibidass** [comment]
we see the same behaviour. **docker_multline** is detecting the Parser_Firstline regex as first line properly. But when we switched to Docker_Mode_Parser, its merging all lines that do not conform to the Parser regex, even if there is no first line present. Implication of this is all application logs that do not confirm to the Parser regex pattern gets merged indefinitely till buffer flush, where as we expect it to output as different lines (since there is no multiline expected out of them).

**Comment by coders-kitchen** [comment]
Here's a minimalistic example to see the issue with a local running fluent bit

**fluent.conf**

    [SERVICE]
     Parsers_File parsers.conf
    
    [INPUT]
     Name tail
     Parser docker
     Path /home/pdaum/fluent-bit/test*.log
     Docker_Mode       On
     Docker_Mode_Parser  docker_multiline
     Refresh_Interval  1
    
    [OUTPUT]
     Name stdout
     Match *

**parsers.conf**

    [PARSER]
     Name docker_multiline
     Format regex
     Regex (?<log>^{"log":"\d{4}-\d{2}-\d{2}.*)
    
    [PARSER]
     Name   json
     Format json
     Time_Key time
     Time_Format %d/%b/%Y:%H:%M:%S %z
    
    [PARSER]
     Name        docker
     Format      json
     Time_Key    time
     Time_Format %Y-%m-%dT%H:%M:%S.%L
     Time_Keep   On

**logproducer.sh**

    #!/bin/bash
    
    log_lines=('{"log":"Line 2\n","stream":"stdout","time":"2020-09-25T07:47:13.096196769Z"}')    
    log_lines+=('{"log":"Line 3\n","stream":"stdout","time":"2020-09-25T07:47:13.096196769Z"}')
    log_lines+=('{"log":"Line 2\n","stream":"stdout","time":"2020-09-25T07:47:13.096196769Z"}')
    log_lines+=('{"log":"Line 3\n","stream":"stdout","time":"2020-09-25T07:47:13.096196769Z"}')
    log_lines+=('{"log":"Line 7\n","stream":"stdout","time":"2020-09-25T07:47:13.096196769Z"}')
    log_lines+=('{"log":"Line 9\n","stream":"stdout","time":"2020-09-25T07:47:13.096196769Z"}')
    #log_lines+=('{"log":"2020-04-03 MULTI-LINE 3\n","stream":"stdout","time":"2020-09-25T07:47:13.096196769Z"}')
    
    for i in `seq 1 100`
    do
      FILE_NUMBER=$((i%5))
      echo ${log_lines[$((RANDOM%9))]} >> test${FILE_NUMBER}.log
    done

**Comment by coders-kitchen** [comment]
The output will be similar to this

    [4] tail.0: [1601020033.096196769, {"log"=>"Line 3
    Line 2
    Line 9
    Line 2
    Line 2
    Line 3
    Line 3
    Line 9
    Line 3
    Line 2
    Line 3
    ", "stream"=>"stdout", "time"=>"2020-09-25T07:47:13.096196769Z"}]

**Comment by sibidass** [comment]
@sumo-drosiek ^^

**Comment by sumo-drosiek** [comment]
@sibidass @coders-kitchen So do you expect that multiline concatenation starts with first valid multiline log?

I don't really understand usecase for that, because:

 a.  if file consists of few logging formats (where some of them are multiline and some of them not) it won't work correctly as it will start to concatenate not-multilline logs anyway
 b. if file contains only multiline logs with well specified format it shouldn't matter unless file is splitted in middle of the log (which end with inconsistent behavior anyway)
 c. if file doesn't contain mulitiline logs by design I don't see a reason to use multiline feature

Only reason to skip concatenation unless first regex match is when someone uses multiline feature for not multiline logs (or regex is invalid), so rather inappropriate configuration 

Anyway, I can add this safety check if this is really needed.


@coders-kitchen for given logs:

```
{"log":"ACCESS [2020-09-25T07:47:03.299Z] \"GET /unleash/api/admin/metrics/feature-toggles HTTP/1.1\" 200 - 0 151 3 2 \"10.244.7.0\" \"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36\" \"686477e2-fb40-447a-9dc9-21334aa816ca\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.09612071Z"}
{"log":"ACCESS [2020-09-25T07:47:05.296Z] \"GET /unleash/api/admin/metrics/feature-toggles HTTP/1.1\" 304 - 0 0 5 3 \"10.244.7.0\" \"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36\" \"e98770d5-836e-48e7-9171-43f74a734a25\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096196769Z"}
{"log":"ACCESS [2020-09-25T07:47:06.030Z] \"GET /unleash/api/client/features HTTP/1.1\" 304 - 0 0 5 4 \"10.244.8.0\" \"whatsappdispatcher\" \"41faa28c-083a-460a-a126-9a9d8e7ed3f4\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096210104Z"}
{"log":"ACCESS [2020-09-25T07:47:07.443Z] \"GET /unleash/api/client/features HTTP/1.1\" 304 - 0 0 6 4 \"10.244.7.0\" \"whatsappdispatcher\" \"c4c38fb5-6553-45e3-a977-eb2cd1ebd9e0\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096246091Z"}
{"log":"ACCESS [2020-09-25T07:47:07.745Z] \"GET /unleash/api/client/features HTTP/1.1\" 304 - 0 0 5 3 \"10.244.7.0\" \"whatsappdispatcher\" \"d6761038-9ab9-4e8f-a558-1491fade3c39\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096256042Z"}
{"log":"ACCESS [2020-09-25T07:47:08.300Z] \"GET /unleash/api/admin/metrics/feature-toggles HTTP/1.1\" 304 - 0 0 3 1 \"10.244.7.0\" \"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36\" \"9b5358aa-7c42-4853-92ac-8f6177f3c19d\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096264898Z"}
{"log":"ACCESS [2020-09-25T07:47:10.297Z] \"GET /unleash/api/admin/metrics/feature-toggles HTTP/1.1\" 304 - 0 0 3 1 \"10.244.7.0\" \"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36\" \"4929e3e5-8146-4832-957f-d6d12cdc3725\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096274135Z"}
{"log":"ACCESS [2020-09-25T07:47:11.030Z] \"GET /unleash/api/client/features HTTP/1.1\" 304 - 0 0 5 3 \"10.244.8.0\" \"whatsappdispatcher\" \"d5da1102-dd20-47cb-8f28-291b1915554e\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096284581Z"}
{"log":"ACCESS [2020-09-25T07:47:12.443Z] \"GET /unleash/api/client/features HTTP/1.1\" 304 - 0 0 5 4 \"10.244.7.0\" \"whatsappdispatcher\" \"6136cb52-56fb-45be-bb4f-31d09e730586\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096293667Z"}
{"log":"ACCESS [2020-09-25T07:47:12.745Z] \"GET /unleash/api/client/features HTTP/1.1\" 304 - 0 0 6 4 \"10.244.7.0\" \"whatsappdispatcher\" \"928911a2-3f3b-4541-b43e-0b09be23c61c\" \"chat-api-int.tech-on-air.com:30080\" \"10.245.208.187:4242\"\n","stream":"stdout","time":"2020-09-25T07:47:13.096302327Z"}

```

I would use regex which catches the given lines properly,eg. `(?<log>^{"log":"ACCESS \[\d{4}-\d{2}-\d{2}.*)`

**Comment by sibidass** [comment]
@sumo-drosiek So the reason why I had to pull back to previous Multiline configuration provided by fluentbit in our kubernetes environment is because of the following reason:
I had a pattern check for logs starting datetime format in Docker_Mode_Parser. This will catch logs with that format. In tail plugin, we use /var/log/containers/*.log which is meant to crawl through logs emitting out of all containers running in the cluster.

We have all kinds of application containers running in a cluster emitting logs in different formats.(I suppose this is the case with most clusters)

Earlier, with Multiline configuration, it will add subsequent lines to first line, provided first line is in proper datetime format. So when tailing a container file, if first line is datetime formatted log, and second line is not, it will identify second line as part of first and merge it accordingly. If the first line itself is not in a datetime format, that means the logs are in a different format and it will be sent as separate line.

Now, once I switched to the new Docker_Mode_Parser, this is not happening. which means , all application logs that do not follow the datetime beginning format are now getting merged indefinitely. we can modify the regex to accomodate other formats, but that is not a practical solution, as we have definite release cycles and we have tons of applications in lots of different formats.

Also both the previous multiline option and the new docker multiline mentions the following in documentation:
Parser_Firstline: Name of the parser that matchs the **beginning** of a multiline message
Docker_Mode_Parser: Specify an optional parser for the **first line** of the docker multiline

So it would be good, if we could seamlessly migrate to docker multiline from fluentbit multiline facility.

**Comment by sibidass** [comment]
btw, I am currently using lua script with fluentbit multiline flush to process multilines properly. it is working fine, but lua is really slow compared to native C and I saw a performance jump after switching to Docker_Mode_Parser, but unfortunately I had to roll back because of noise from users that their logs are getting merged.

**Comment by sumo-drosiek** [comment]
@sibidass so expected behavior for you is to start concatenating multlines from the first match of the `Docker_Mode_Parser` or only if the first line of the log matches the given regex?

**Comment by sibidass** [comment]
@sumo-drosiek I believe concatenating multilines from first match is good enough, which will make Docker_Mode_Parser fully compatible with the multiline mode on, as I could see that is the current behaviour of fluentbit multiline flush.

**Comment by sumo-drosiek** [comment]
@coders-kitchen Will such solution solve your issue as well?

**Comment by coders-kitchen** [comment]
Yes, that would perfectly match our expectations as well

**Comment by sumo-drosiek** [comment]
@sibidass @coders-kitchen The fix is included in the last fluent-bit version: [v1.6.0](https://github.com/fluent/fluent-bit/releases/tag/v1.6.0) 🎉

**Comment by sibidass** [comment]
> @sibidass @coders-kitchen The fix is included in the last fluent-bit version: [v1.6.0](https://github.com/fluent/fluent-bit/releases/tag/v1.6.0) 🎉

wow nice. I will test this out. 🧪
